bin/process_h5ad.py

```diff
@@ -15,7 +16,16 @@ def h5ad_to_zarr(
     compute_embeddings=False,
     chunk_size=10,
     ):
+    #check if index is integer, if not reindex
 
+    if not adata.obs.index.is_integer(): 
+        adata.obs['spot_id'] = adata.obs.index
```

Comment:
could you please change `spot_id` to something more general like `label_id`? as it can contain cells instead of spots